packages/wp-now/src/tests/wp-now.spec.ts

```diff
@@ -144,3 +151,107 @@ test('isWordPressDevelopDirectory returns false for incomplete WordPress-develop
 	expect(isWordPressDevelopDirectory(projectPath)).toBe(false);
 	expect(inferMode(projectPath)).toBe(WPNowMode.INDEX);
 });
+
+describe('Test starting different modes', () => {
+	let tmpExampleDirectory;
+
+	/**
+	 * Copy example directory to a temporary directory
+	 *
+	 * Also, silent the console.log() output.
```

Comment:
Maybe we should have a helper function for our log output, so we don't have to suppress the entirety of `console.log()` in tests?

Reply:
Do you mean adding a condition on the implementation side to use `console.log()` only if the code is executed outside the test context?

Reply:
@wojtekn Yes.

Reply:
I've checked it more, and I found out that `jest` has a `--silent` option built-in. We can run this one for CI and normal test execution, and skip the flag if we need to see the output, e.g. for test debugging purposes.

```
nx test wp-now --silent
```

That way we wouldn't need to suppress it in tests code, and we wouldn't need to hardcode hiding output for jest context in the tool code.